src/anls.jl

```diff
@@ -1,31 +1,33 @@
 module ANLS
 """
 Fit CNMF using Alternating Non-Negative Least Squares.
-Note: this requires having the NonNegLeastSquares source available 
-in a directory adjacent to the cmf.jl directory.
 """
 
 # Import 
-import NonNegLeastSquares
+using NonNegLeastSquares
```

Comment:
This is imported in CMF.jl – so we shouldn't need to import it again.

Reply:
Actually it's not at the moment. Additionally, this is within the module `ANLS`, which is the only place where `NonNegLeastSquares` is used. There might be namespace problems if we do the import outside the module, but either way I think it makes the most sense to keep the import in this file if it's only used here.

Reply:
ah you are right! my bad.